src/System.Security.Cryptography.Pkcs/tests/CertLoader.cs

```diff
@@ -41,7 +41,11 @@ public X509Certificate2 TryGetCertificateWithPrivateKey()
                 // and having our test certificates there can cause false test errors or false test successes.
                 //
                 // This may be an odd place to do this check but it's better than expecting every individual test to remember to do this.
-                CheckIfCertWasLeftInCertStore();
+                try
+                {
+                    CheckIfCertWasLeftInCertStore();
+                }
+                catch (PlatformNotSupportedException) { }
```

Comment:
This try/catch should just be around the `FindMatches("MY", StoreLocation.LocalMachine);` call